themes/digital.gov/src/js/guide-mobile-menu.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Display horizontal desktop or vertical mobile menu when viewport size changes
+ *
+ */
+
+const dgGuideMenuBar = document.querySelector(".dg-guide__menu-bar");
+const dgAccordionMobileMenu = document.querySelector(".guide-mobile-menu");
+const dgAccordionContent = document.querySelector(
+  ".guide-mobile-menu .usa-accordion__content"
+);
```

Comment:
You have the mobile menu, you could query that instead of the document. This is faster than querying the entire DOM again.


```suggestion
const dgAccordionContent = dgAccordionMobileMenu.querySelector(".usa-accordion__content");
```

Reply:
Good catch!